frontend/cypress/e2e/beta/draft-and-review-release.cy.ts

```diff
@@ -0,0 +1,94 @@
+const baseURLForRelease = '/beta/'
+let modelUuidForRelease = ''
+const modelNameForRelease = 'Test Model'
+const releaseVersion = '1.0.0'
+
+describe('Draft and review a model release', () => {
+  before(() => {
+    cy.log('Upload new model and set schema via API')
+    cy.request('POST', 'http://localhost:8080/api/v2/models', {
+      name: modelNameForRelease,
+      teamId: 'Uncategorised',
+      description: 'This is a test',
+      visibility: 'public',
+    }).then((response) => {
+      expect(response.status).to.eq(200)
+      expect(response.body.model).to.have.property('name', modelNameForRelease)
+      modelUuidForRelease = response.body.model.id
+      cy.request('POST', `/api/v2/model/${modelUuidForRelease}/setup/from-schema`, {
+        schemaId: 'minimal-general-v10-beta',
+      }).then((response) => {
+        expect(response.status).to.eq(200)
+        expect(response.body.card).to.have.property('modelId', modelUuidForRelease)
+      })
+    })
+  })
+
+  it('drafts a new release for a model', () => {
+    cy.visit(`${baseURLForRelease}/model/${modelUuidForRelease}`)
+    cy.contains(modelNameForRelease)
```

Comment:
I always prefer to check that an element we expect to be present contains some content we expect to also be present. It feels more robust than checking the entire page for text. Consider updating to ensure all `cy.contains` checks have a corresponding `cy.get`